premake5.lua

```diff
@@ -4,6 +4,8 @@ include "premake5_helpers.lua"
 include "Tests/premake5.lua"
 include "Tests/CSharp/premake5.lua"
 include "Tests/Multiplayer/premake5.lua"
+include "MultiplayerTestRunner/premake5.lua"
+include "MultiplayerTestManager/premake5.lua"
```

Comment:
[fix] Don't leak test manager stuff in this PR

Reply:
https://github.com/magnopus-opensource/connected-spaces-platform/pull/558/commits/b8f88a1d5867389f340ffbd7135818ed4b4fd7f5